src/stores/defaults.ts

```diff
@@ -51,7 +51,10 @@ export const storeStateDefault: StoreState = {
     }
   ],
   currentMatchIndex: 1,
-  matchIdToMatchState: {},
+  matchIdToMatchState: {
```

Comment:
revert

Reply:
Yes, you don't want these defaults to change. You want to use tests to test your reducer.